state/pgstatestorage/pgstatestorage.go

```diff
@@ -128,7 +128,7 @@ func (p *PostgresStorage) GetLogsByBlockNumber(ctx context.Context, blockNumber
        INNER JOIN state.l2block b ON b.block_num = t.l2_block_num
        INNER JOIN state.receipt r ON r.tx_hash = t.hash
        WHERE b.block_num = $1
-       ORDER BY l.log_index ASC`
+       ORDER BY r.tx_index, l.log_index ASC`
```

Comment:
is it no better to indicate explicitly that we want to sort in  ASC order the tx_index field?